src/App.js

```diff
@@ -1,9 +1,15 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { FormattedHTMLMessage } from 'react-intl';
+import { addLocaleData } from 'react-intl';
```

Comment:
Why not using these both lineas in only one?

```js
import { FormattedHTMLMessage, addLocaleData } from 'react-intl';
```

Is it a desired coding style? or it is only something casual?

Reply:
Casual 😛